code/modules/mob/living/brain/brain_item.dm

```diff
@@ -37,7 +37,7 @@
 		if(C.has_brain_worms())
 			var/mob/living/simple_animal/borer/B = C.has_brain_worms()
 			B.leave_victim() //Should remove borer if the brain is removed - RR
-		transfer_identity(C)
+	transfer_identity(C)
```

Comment:
Should that be in a separate pull or is atomization more a thing for features? Also, fixing surgical removal would allow a changeling test by cutting out the brain: the examine tests whether the individual has a brain or not, so when a changeling keeps their brain and gets examined they'll clearly not be brainless.